aixcalibuha/utils/visualizer.py

```diff
@@ -360,15 +348,22 @@ def __init__(self, cd,
                          name=name,
                          calibration_class=calibration_class,
                          logger=logger)
+
+        # Setup dummy parameters so class-functions
+        # now the type of those later created objects:
+        self._n_cols_goals, self._n_rows_goals, self._n_cols_tuner, self._n_rows_tuner = 1, 1, 1, 1
+        self.fig_tuner, self.ax_tuner = None, None
+        self.fig_goal, self.ax_goal = None, None
+        self.fig_obj, self.ax_obj = None, None
+        self._num_goals = 0
+        self.goals_dir = "TimeSeriesPlot"
         # Set supported kwargs:
-        if isinstance(kwargs.get("save_tsd_plot"), bool):
-            self.save_tsd_plot = kwargs.get("save_tsd_plot")
-        if isinstance(kwargs.get("create_tsd_plot"), bool):
-            self.create_tsd_plot = kwargs.get("create_tsd_plot")
-        if isinstance(kwargs.get("show_plot"), bool):
-            self.show_plot = kwargs.get("show_plot")
-        if isinstance(kwargs.get("file_type"), str):
-            self.file_type = kwargs.get("file_type")
```

Comment:
I see why removing the type check for these kwargs is not necessary. However, for `show_plot_pause_time ` the user could potentially use a string, which would break the code without a good error message (time.sleep raises a typeError, but to a new user its not clear which kwarg might influence this). Maybe adding the type check for this kwarg would help. But this is just a small detail. Going to approve the PR. Thanks!

Reply:
I added the check, thanks!